dashboard/lib/constants/tremor-colors.ts

```diff
@@ -0,0 +1,32 @@
+import { Color } from '@tremor/react'
+
+// https://www.tremor.so/docs/layout/color-palette
+const tremorColors: Record<Color, string> = {
```

Comment:
We have the theme file [`/web-analytics-starter-kit/dashboard/styles/theme.js`](https://github.com/tinybirdco/web-analytics-starter-kit/blob/main/dashboard/styles/theme.js) where we wanted to centralized all the application colors to be able to modify the UI theme just changing one single file and one single line.

I think we should keep using the primary color to create the graphs, to keep being able to customize the dashboard just modifying the `primary` color constant. Otherwise, the users would have to modify the color Widget by Widget. WDYT?

Reply:
These mustn't be changed, since these are the only ones allowed in tremor props. One thing that could be done is to refer to these colors in the theme `{ ..., primary: tremorColors["blue"] }`.

Reply:
OK! I see your point.

So, as an intermediate solution, I would move this and the other file to a common folder like `/theme` containing `/theme/index.js` with the [theme.js](https://github.com/tinybirdco/ web-analytics-starter-kit/blob/main/dashboard/styles/theme.js) content and `/theme/tremor-colors.ts` file

Just to have all this config in a single place, and the users doesn't have to look for it. 

And we would need update the [Customization](https://github.com/tinybirdco/web-analytics-starter-kit/tree/main/dashboard#customization) section of the project adding that if you want to customize the chart colors you would have to change the value one by one in the charts components.